_posts/2025-04-24-Announcing-Data-Prepper-2.11.md

```diff
@@ -0,0 +1,276 @@
+dramatically enhancing information accessibility and utilization within the organization. Here's how to get started:
+
+```
+version: "2"
+extension:
+    aws:
+      secrets:
+        confluence-account-credentials:
+          secret_id: <<secret-arn>>
+          region: <<secrets-region>>
+          sts_role_arn: <<role-to-access-secret>>
+confluence-pipeline:
+  source:
+    confluence:
+      hosts: ["<<Atlassian-host-url>>"]
+      authentication: 
+        basic:
+          username: ${{aws_secrets:confluence-account-credentials:confluenceId}}
+          password: ${{aws_secrets:confluence-account-credentials:confluenceCredential}}
+      filter:
+        space:
+          key:
+            include:
+              - "<<space-key>>"
+            exclude:
+               - "<<space-key>>"
+        page_type:
+          include: 
+            - "page"
+            - "blogpost"
+            - "comment"
+          exclude:
+            - "attachment"
+  sink:
+    - opensearch:
+
+```
+
+
+## Amazon Aurora/RDS as a source
+
+[Amazon Aurora](https://aws.amazon.com/rds/aurora/) and [Amazon RDS](https://aws.amazon.com/rds/) are fully managed relational database services that make it easier to set up, operate, and scale a relational database in the AWS Cloud.
+For those who want to take advantage of advanced search capabilities like full-text and vector search on the transactional data in Amazon Aurora/RDS, you can now use Data Prepper to synchronize data from Aurora and RDS to OpenSearch.
+
+Data Prepper’s new `rds` source first exports existing data from Amazon Aurora/RDS tables to OpenSearch indices, then streams incremental changes from those tables to keep the data consistent between the relational database and OpenSearch.
+The `rds` source currently supports Aurora MySQL, Aurora PostgreSQL, RDS MySQL, and RDS PostgreSQL engines.
+
+
+The following is an example configuration:
+
+```yaml
+aurora-mysql-pipeline:
+  source:
+    rds:
+      db_identifier: "my-aurora-cluster"
+      engine: "aurora-mysql"
+      database: "hr_db"
+      tables:
+        include:
+          - "employees"
+          - "departments"
+      s3_bucket: "my-s3-bucket"
+      s3_prefix: "pipeline-data"
+      export:
+        kms_key_id: "1234abcd-1234-abcd-1234-123456abcdef"
+        export_role_arn: "arn:aws:iam::123456789012:role/ExportRole"
+      stream: true
+      aws:
+        sts_role_arn: "arn:aws:iam::123456789012:role/PipelineRole"
+        region: "us-east-1"
+      authentication:
+        username: ${{aws_secrets:secret:username}}
+        password: ${{aws_secrets:secret:password}}
+```
+
+## SQS as a source
+
+Data Prepper now supports a new Amazon Simple Queue Service (SQS) source for reading events from SQS queues.
+Amazon SQS is a fully managed message queue.
+The new SQS source in Data Prepper efficiently receives messages from SQS to create events you can route to sinks such as OpenSearch.
+
```

Comment:
Line 218: "`sqs` source"? Please combine the three lines into one.

Reply:
I changed to `sqs` source.